Doc/library/os.rst

```diff
@@ -1730,6 +1730,11 @@ features:
 
    Return a bytestring representing the current working directory.
 
+   .. versionchanged:: 3.9
```

Comment:
Why not fix it in 3.8 as well? It's not a new feature

Reply:
It's a backward incompatible change.